src/configuration.ts

```diff
@@ -63,6 +72,11 @@ export type UserMessages = {
 // 🛠
 
 
+export function addRootFileSystemPermissions(config: Configuration): Configuration {
+  return { ...config, permissions: { ...config.permissions, ...ROOT_FILESYSTEM_PERMISSIONS } }
+}
```

Comment:
Is this something that developers would use in their apps? Not seeing it used in our code.

Reply:
Ah right, don't need this anymore, I'll remove it.